ipamd/ipamd.go

```diff
@@ -207,21 +220,22 @@ func (c *IPAMContext) StartNodeIPPoolManager() {
 	for {
 		time.Sleep(ipPoolMonitorInterval)
 		c.updateIPPoolIfRequired()
+		c.nodeIPPoolReconcile(nodeIPPoolReconcileInterval)
```

Comment:
Is there a reason this reconcile loop doesn't run in it's own go routine, and just uses the same approach with a sleep like the update loop? Then there would be no need to set and check against `lastNodeIPPoolAction`.

Reply:
The intend of ```lastNodeIPPoolAction``` is to make sure it does NOT do reconcile right after  increasing or decreasing the IP Pool, so it is needed even it is put inside ```c.updateIPPoolIfRequred()```

Reply:
Ah, that makes sense. Forgot that `updateIPPoolIfRequired()` also updates that time.